68513 IU5- Slow posting time

tyndmar opened the pull request:
File changed:
fg/fgPostBatch.p
- replaced REPEAT loop with DO WHILE with embedded 1 second pause
- added user messaging to advise of issue encountered

Commit:
68513 IU5- Slow posting time

File changed:
fg/fgPostBatch.p
- replaced REPEAT loop with DO WHILE with embedded 1 second pause
- added user messaging to advise of issue encountered

jay-advantzware reviewed (commented):
NO - We do not want to lock FGItem at all nor send a message.  Do a no wait, no error and if item fg cannot be updated, skip the update and proceed

jay-advantzware reviewed (commented):
Should code at line 666 be removed as duplicate code?

jay-advantzware reviewed (commented):
Can we release before sending email (If setup)?

Commit:
68513 IU5 Slow Posting Time

file changed: fg/fgPostBatch.p
added timer to limit number of DB reads and network packets

jay-advantzware reviewed (commented):
Looks good.
